prql-compiler/src/parser/chumsky.rs

```diff
@@ -78,3 +89,37 @@ pub fn ident() -> impl Parser<char, Ident, Error = Simple<char>> {
         .chain(just('.').ignore_then(ident_part().or(star)).repeated())
         .map(Ident::from_path::<String>)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    use insta::assert_yaml_snapshot;
+    #[test]
+    fn test_date() {
+        let date = just('@')
+            .ignore_then(digits(4))
+            .then_ignore(just('-'))
+            .then(digits(2))
+            .map(join::<'-'>)
+            .then_ignore(just('-'))
```

Comment:
Not a huge deal — maybe we should just proceed and then come back and refine — but it's seems repetitive to have the `.then_ignore(just('-'))` and then `join::<'-'>`.

@zesterer feel free to ignore, but let us know if you see this + there's a canonical chumsky solution...

Reply:
Super, thanks a lot @zesterer